public/static/js/legacyMain.js

```diff
@@ -80,14 +80,40 @@ const listenScrollToTopButton = () => {
   });
 };
 
-const detectEnviromentAndSetDownloadOptions = () => {
+const detectEnviromentAndSetDownloadOptions = async () => {
```

Comment:
Let's not make this asynchronous.

Reply:
If not, the awaitable method cannot be called....., and the compiler will report an error, because "userAgentData.getHighEntropyValues" is an async method, which must be called by "await" in an async method.